synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -131,26 +131,28 @@ class studio::StateBone_Context : public sigc::trackable
 
 	void set_active_bone(ValueNode::Handle value_node);
 
+	static constexpr Real default_angle = 3.14159265358979323846/2;
```

Comment:
Is there a reason not to use `M_PI` here? I am asking because I planned to replace all these values with named definitions/constants.

Reply:
Yes: I just forgot it :)